app/views/events/show.html.erb

```diff
@@ -0,0 +1,76 @@
+<style>
+  .details { color: #333; }
+  .mic_row { display: flex }
+  .mic_row > div { padding: 10px; }
+  .when { color: #777; font-size: 120%; }
+  .attendees > a
+  {
+    display: block;
+    width: 8em;
+    height: 8em;
+    border: 1px solid #888;
+    border-radius: 4px;
+    float: left;
+    margin: 1em;
+    padding: 1em;
+    background-color: rgb(238, 238, 238);
+    box-sizing: border-box;
+    text-decoration: none;
+  }
+</style>
```

Comment:
Inline CSS is not allowed so this needs to be moved to a stylesheet.

Reply:
Yes.  My first attack at groups (2018) used SASS (https://github.com/openbrian/microcosm-aoi/blob/master/app/assets/stylesheets/application.css.scss#L141 notice different repo here).  Is that ok?

Reply:
Yes SASS is fine and it can either go in there or you can create a new file in that directory.

It just can't be inline because our content security policy prohibits that. I'm surprised you didn't encounter that when testing to be honest.